index.js

```diff
@@ -14,7 +14,7 @@ const Peer = require('simple-peer')
 const randombytes = require('randombytes')
 const speedometer = require('speedometer')
 const queueMicrotask = require('queue-microtask')
-
+require('./hacks')
```

Comment:
I think the maintainers will not approve, lol

Reply:
Probably not! I didn't intend on leaving it this way.